Sources/ProjectSpec/Scheme.swift

```diff
@@ -58,8 +58,13 @@ public struct Scheme: Equatable {
 
     public struct Test: BuildAction {
         public var config: String
+        public var codeCoverageEnabled: Bool
         public init(config: String) {
+            self.init(config: config, codeCoverageEnabled: false)
+        }
+        public init(config: String, codeCoverageEnabled: Bool) {
```

Comment:
Looking at the `BuildAction` protocol it doesn't seem we don't actually use the init anywhere, so we can remove that and merge these 2 inits
```
public init(config: String, codeCoverageEnabled: Bool = false) {
```

Reply:
That was my first approach but...
`Type 'Scheme.Test' does not conform to protocol 'BuildAction'`
😭 😭 😭

Reply:
Yeah, I mean we can remove the init from the `BuildAction` protocol, then it won't complain anymore. We're not using that protocol conformance anywhere anyway

Reply:
Because the other build actions will surely have other injected properties in the future. The fact these are mutable means we don't HAVE to put properties in the inits, but it's just convenient (for tests and anyone using this as a framework)

Reply:
I understand, change made. 👍